src/index.tsx

```diff
@@ -189,10 +189,11 @@ initialize({
       component: () => (
         <MapApp
           logoProps={{
-            href: 'https://veupathdb.org',
-            src:
+            siteHomeUrl: 'https://veupathdb.org',
+            siteLogoSrc:
               'https://veupathdb.org/veupathdb/images/VEuPathDB/icons-footer/vectorbase.png',
             siteName: 'VectorBase',
+            loginUrl: 'https://eupathdb.org/oauth/assets/eupathdb-login.html',
```

Comment:
FYI: These props are used to render the logo in the map header as well as the site URL and login URL in the side menu.